sdk/monitor/Azure.Monitor.OpenTelemetry/README.md

```diff
@@ -1,19 +1,53 @@
 # Azure Monitor Distro client library for .NET
 
+The Azure Monitor Distro is a client library that sends telemetry data to Azure Monitor following the OpenTelemetry Specification. This library can be used to instrument your ASP.NET Core applications to collect and send telemetry data to Azure Monitor for analysis and monitoring, powering experiences in Application Insights.
```

Comment:
the "Key Concept" content could be moved here.
I dont see any strong value in having "Key Concept" section.

Reply:
CI does not allow to move the sections. We could use the same content as of exporter in key concepts.

https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter#key-concepts